### PR TITLE
Blog post: Qrack and PyQrack join the Unitary Fund Organization

### PR DESCRIPTION
**Pull request by WrathfulSpatula**
Qrack and PyQrack join the Unitary Fund Organization! Read more inside!

### CODE REVIEW

File: blog.html [hunk 1]
```diff
@@ -71,6 +71,12 @@
               </p>
               <section class="light">
                 <ul style="list-style-type: none">
+                <li class="leading-block">
+                    <span class="blog-post-date">21 July, 2022</span>
```
Comment (natestemen) [inline blog.html:75]: date will have to change (as you mentioned on discord). Just flagging for visibility.

File: posts/qrack_joins_uf.html [hunk 2]
```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+<script src="https://kit.fontawesome.com/22ffc91bdd.js" crossorigin="anonymous"></script>
+
+<head>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset="utf-8">
+    <meta name="date" content='2022-05-24'>
```
Comment (natestemen) [inline posts/qrack_joins_uf.html:9]: date

File: posts/qrack_joins_uf.html [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+                                <p>Started by Daniel Strano (technical staff member of Unitary Fund) and Benn Bollay (CTO of Fusebit) in 2017 and winner of a Unitary Fund microgrant in 2019, Qrack is a quantum computer gate model simulator written in C++11. Qrack is intended to provide maximum performance and minimum memory footprint for quantum computing workloads run on “classical” computer hardware. PyQrack is Qrack’s dependency-free pure Python “ctypes” wrapper, to expose Qrack shared library binaries directly for just-in-time (JIT) execution via a Python interpreter. Plugins and providers are available for Qiskit, XACC, and even the Unity video game engine, with plans to expand plugin support to virtually every major quantum open source framework in the global ecosystem. Supported platforms include all available OEM combinations of x86_64, x86, ARMv7, and ARM64 instruction sets with Linux, Windows, Mac, iOS, and Android operating systems, as well as WebAssembly (Wasm).</p>
```
Comment (natestemen) [inline posts/qrack_joins_uf.html:81]: 1. why `"classical"`?
2. maybe "...dependency-free Python "ctypes wrapper..." (remove pure)
3. Can you add links to Qiskit, XACC, and maybe unity?
4. What is OEM?

File: posts/qrack_joins_uf.html [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+                                <p style="font-weight: bold"><strong>QFT Benchmark</strong></p>
+                                <p>A standard example of Qrack (and PyQrack) performance is the quantum Fourier transform (QFT). Comparative results from benchmarking are in the graph below and are <a href="https://metriq.info/Task/142">available on Metriq</a> (as single-valued exponential trendline slope comparisons). Historically, Qrack measures this benchmark with an initialization by random 3-parameter unitary single qubit gates across the register width for a fair comparison, because Qrack QFT performance on computational basis states is classically efficient because of the underlying Schmidt decomposition. That wouldn’t be representative of general demands for cases of the QFT. While Qrack would perform worse for a random fully entangled case, our choice of randomly initialized separable qubits generally become fully entangled by the end of the circuit, paying a representative performance penalty for potentially “composing” every initially separable qubit. Hence this has historically been deemed a “fair” benchmark by the Qrack team.<p>
```
Comment (natestemen) [inline posts/qrack_joins_uf.html:84]: remove quotes around `fair`?

File: posts/qrack_joins_uf.html [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+                                <p>PyQrack has a commanding performance lead on this benchmark compared to other popular quantum computer simulators like Qiskit Aer (GPU) and QCGPU, where these three candidates are GPU-based simulators. Qiskit Aer (CPU) and Cirq (CPU) have been included on the chart, run on the same stock gaming laptop as the three GPU simulators, but remember that these software benchmarks are most fairly compared on identical hardware, whereas it isn’t possible for a CPU model and a GPU model to be the exact same hardware instance, obviously. However, PyQrack’s “hybrid CPU/GPU” methods utilize CPU simulation at the low qubit width end of the trend, which should be a fair comparison against CPU simulators until the user-selectable GPU switching threshold, which is 9 qubits in this case.</p>
```
Comment (natestemen) [inline posts/qrack_joins_uf.html:85]: I wonder if the image should go after this paragraph?

File: posts/qrack_joins_uf.html [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+                                <p>Qrack is a collection of stacking and interchangeable optimization “layers,” which include OpenCL-based, scalable multi-device GPU, CPU, and GPU/CPU “hybrid” state vector simulation at base, “hybrid stabilizer” simulation with full and transparent recourse to state vector representation as fallback, an auxiliary “quantum binary decision tree” layer, and Schmidt decomposition techniques layered over stabilizer and state vector simulation for proactively and reactively identifying separable “cuts” in qubit subsystem representations to reduce memory footprint and improve execution speed. Floating point precision options available from source build include half, float, and double, with even limited support for quad math. By design, most users don’t need to worry about complicated custom configuration of the layers and settings at all; just instantiate a default C++11 “QInterface” or Python “QrackSimulator” class and go! The highly-optimized default layer stack will automatically leverage hybrid stabilizer simulation for a Clifford circuit or preamble, for example.</p>
```
Comment (natestemen) [inline posts/qrack_joins_uf.html:87]: quotes around `layers` probably shouldn't encapsulate the comma, and I'm not sure they are needed.

File: posts/qrack_joins_uf.html [hunk 7]
```diff
@@ -0,0 +1,169 @@
...
+                                <p><img src="../images/qrack_qft_full_width.png"/><p>
+                                <p><span style="font-weight: bold">Check out the repositories</span> on the Unitary Fund GitHub organization, “star” and share, and get Qrackin’! You rock!</p>
```
Comment (natestemen) [inline posts/qrack_joins_uf.html:89]: Add the links!

**Review by natestemen — COMMENTED** [review]
Big congratulations Dan, this is very exciting!

A few comments, not all of which need addressing, but you do seem to use a lot of quotes where I'm not sure they are necessary.

Also for future HTML PR's can you shorten the line length, or use something like one sentence per line to make the review easier.

**Comment by willzeng** [comment]
Looks like a few remaining suggestions here @WrathfulSpatula Let me know when ready for re-review!

File: posts/qrack_joins_uf.html (hunk 4 above)
Comment (willzeng) [reply posts/qrack_joins_uf.html:84]: +1

File: posts/qrack_joins_uf.html (hunk 6 above)
Comment (willzeng) [reply posts/qrack_joins_uf.html:87]: can remove the quotes

File: posts/qrack_joins_uf.html (hunk 7 above)
Comment (willzeng) [reply posts/qrack_joins_uf.html:89]: and can remote the quotes around "star"

File: posts/qrack_joins_uf.html (hunk 5 above)
Comment (WrathfulSpatula) [reply posts/qrack_joins_uf.html]: @willzeng What do you think? I thought the same thing as Nate.

**Comment by WrathfulSpatula** [comment]
@willzeng I addressed everything except the dates and the one conversation I tagged you in above, whenever you want to take another look.

**Comment by nathanshammah** [comment]
Thanks @WrathfulSpatula! LGTM. When this is approved by @willzeng, and merged, please get in touch with Alowina and Frances for social media posts.

**Comment by willzeng** [comment]
Two quick things:
[1] Let's put this line at the end of the first paragraph in highlighter yellow: "Get started with Qrack from the docs [here](https://qrack.readthedocs.io/en/latest/)."
[2] Worth a quick mention of the license for the repo in the second paragraph.

Am approving!

**Review by willzeng — DISMISSED** [review]
Two quick things:
[1] Let's put this line at the end of the first paragraph in highlighter yellow: "Get started with Qrack from the docs [here](https://qrack.readthedocs.io/en/latest/)."
[2] Worth a quick mention of the license for the repo in the second paragraph.

Am approving!

@WrathfulSpatula

**Comment by WrathfulSpatula** [comment]
@willzeng Pushing your requested changes rendered the review stale. I also changed the date, to publish today, but I need approval, please.

**Review by willzeng — APPROVED** [review]
Lgtm

**Pull request merged**